tools/explorer/lib/parse_input.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Main module for DBO explorer."""
+import ast
+from termcolor import colored
+from typing import List
+
+from lib import model
+from lib.model import StandardField
+
+DEFAULT_MATCHED_TYPES_LIST_SIZE = 10
+
+def _InputFieldsFromUser(ontology, manual: bool = False) -> List[StandardField]:
+  """Method to take in field inputs from the user
+
+  stuff
```

Comment:
I think you forgot to complete the docstring?

Reply:
thank you for combining those

Reply:
Whoops....